sway/tree/workspace.c

```diff
@@ -343,3 +385,13 @@ bool workspace_switch(struct sway_container *workspace) {
 	arrange_windows(output, -1, -1);
 	return true;
 }
+
+bool workspace_is_visible(struct sway_container *ws) {
+	struct sway_container *output = container_parent(ws, C_OUTPUT);
+	struct sway_seat *seat = input_manager_current_seat(input_manager);
+	struct sway_container *focus = sway_seat_get_focus_inactive(seat, output);
```

Comment:
There's an ugly situation where this fails.

The focus inactive is the last focused container that is a child of the output. What if that window closes? The window that was focused before that might be on a different workspace.

Focus inactive should revert to another window on the workspace in that case.

Reply:
Sounds like a separate ticket?

Reply:
Yeah this function is conceptually correct.

Reply:
#1670